### DISCUSSION TITLE
Mouse cursor corruption using ATI Graphics Pro Turbo (Mach64)

### DISCUSSION THREAD

**Discussion by ArnoldUK** [86Box/86Box#2394] [General]
Edit: This is only an issue with V2.x builds.
Mouse cursor corruption using ATI Graphics Pro Turbo (Mach64) in Windows 3.1 and 95. This happens with all driver versions and resolutions including default Windows 95 driver. I have a hunch it is something to do with hardware/software cusror but no idea how to change from hardware to software cursor.
See my GitHub Page: [https://github.com/ArnoldUK](https://github.com/ArnoldUK)

**Comment by TC1995** [comment]
It's not there in the latest builds.

**Reply by ArnoldUK** [reply]
What isn't ?

**Reply by TC1995** [reply]
The bug, 2.07 is too old.

**Comment by ArnoldUK** [comment]
Oh, sorry, yeah you are correct. fixed in 3.0 builds. I'm using v2.07 because I cant the mouse to work in any of the 3.0 builds.

**Reply by TC1995** [reply]
3.0 is too old too.

**Reply by ArnoldUK** [reply]
Yeah I know but in 3.x builds the PS/2 mouse is not working in DOS or Windows and serial/bus mouse are not even detected when installing windows. I'm using 2.07 because the mouse works out of the box.

The ATI Pro Turbo is the only graphic card that gives really good performance in Windows. All the others have very slow re-draw and the mouse is very stuttery.

**Comment by ArnoldUK** [comment]
I've found a work around for the mouse pointer issue in build 2.07.
Setting the mouse pointer scheme to 3D pointers. All other schemes have the corruption bug.
It's a strange fix but works!

**Reply by TC1995** [reply]
it's because you're using a software cursor then, not a hardware cursor.

**Comment by ArnoldUK** [comment]
I've no idea and don't know where to set the type of cursor used by the driver. I simply chanegd the pointer scheme and only the 3D pointers worked without the artifacts in the screenshot. I know some drivers provide a setting in either an ini file or the registry for the cursor but the ATI pro Mach64 does not have an option for the cursor mode in system.ini
